odl/discr/discr_mappings.py

```diff
@@ -771,10 +776,10 @@ def __call__(self, x, out=None):
                 raise ValueError('output shape {} not equal to expected '
                                  'shape {}'.format(out.shape, out_shape))
 
-        indices, norm_distances = self._find_indices(x)
+        indices, norm_distances = self.__find_indices(x)
```

Comment:
This method could actually be worth having non-protected to allow subclasses to implement more efficient variants.